tests/unit/autojac/test_mtl_backward.py

```diff
@@ -105,72 +99,13 @@ def test_mtl_backward_single_task():
     mtl_backward(
         losses=[y1],
         features=[r1, r2],
-        tasks_params=[[p1]],
-        shared_params=[p0],
         A=UPGrad(),
     )
 
     for p in [p0, p1]:
         assert (p.grad is not None) and (p.shape == p.grad.shape)
 
 
-def test_mtl_backward_incoherent_task_number():
```

Comment:
I think we should add back these tests as we have a mix between the new and old versions.

Reply:
Yes, if we manually specify wrong params, let's still check that an error is raised